spring-integration-file/src/main/java/org/springframework/integration/file/filters/ChainFileListFilter.java

```diff
@@ -31,11 +32,25 @@
  *
  * @author Artem Bilan
  * @author Gary Russell
+ * @author Cengis KOCAURLU
```

Comment:
Do you really always write your last name with only capital letters?
Can't it be used just formal as all others?

Reply:
Sorry, it's just an old habit